### PR TITLE
Adds FAKE task for migrations

### PR DESCRIPTION
**Pull request by discorick**
Scripts out the db migrations...

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -59,6 +59,34 @@ Target "Docker" (fun _ ->
         info.Arguments <- "build -t cache ."
         info.WorkingDirectory <- ".") (System.TimeSpan.FromMinutes 5.0) |> ignore
 )
+
+Target "Migrations" (fun _ ->
+    ExecProcess (fun info ->
```
Comment (drusellers) [inline build.fsx]:
```
!!("./migrations/up/*.sql")  |> Seq.iter (fun sqlFile -> 
    ExecProcess (fun info ->
        info.FileName <- "psql"
        info.Arguments <- (sprintf "-U postgres -f %s" file)
        info.WorkingDirectory <- "./migrations/up") (System.TimeSpan.FromMinutes 5.0) |> ignore
) |> ignore
```

Or something like that should clean that up quite a bit. :)

Comment (discorick) [reply build.fsx]: Oooh nice, I was actually going to look into adding an iterator instead, Ill def make that change

**Comment by discorick** [comment]
This should be good to merge now!

**Review by drusellers — APPROVED** [review]

**Comment by drusellers** [comment]
I merged it - but it was ugly - and now i'm sad. but its there. :)

**Comment by discorick** [comment]
> I merged it - but it was ugly - and now i'm sad. but its there. :)

Agreed, it is ugly...  I will go back and refactor when I have a better handle on the language!

**Pull request closed without merging**